File: modul 6/python/PRAK603-2310817210027-MuhammadErzaRaihan.py
row = 2
banyak_row1, banyak_row2 = map(int, input().split())

if banyak_row1 == banyak_row2:
    elemen = [[0] * banyak_row2 for _ in range(row)]

    for i in range(row):
        input_elemen = input().split()
        for j in range(banyak_row1):
            elemen[i][j] = int(input_elemen[j])

    for i in range(row - 1):
        for j in range(banyak_row1):
            if i + 1 < row:
                elemen[i][j] *= elemen[i + 1][j]

    for i in range(row - 1):
        for j in range(banyak_row1):
            print(elemen[i][j], end=" ")
else:
    print("Jumlah tidak sama")

File: modul 6/c/PRAK603-2310817210027-MuhammadErzaRaihan.c
#include <stdio.h>

int main()
{
    int row, banyak_row1, banyak_row2;
    row = 2;
    scanf("%d %d", &banyak_row1, &banyak_row2);
    if (banyak_row1 == banyak_row2)
    {
        int elemen[row][banyak_row2]; 

        for (int i = 0; i < row; i++)
        {
            for (int j = 0; j < banyak_row1; j++)
            {
                scanf("%d",&elemen[i][j]);
            }
        }
        for (int i = 0; i < row-1; i++)
        {
            for (int j = 0; j < banyak_row1; j++)
            {
                if (i + 1 < row) {
                    elemen[i][j] = elemen[i][j] * elemen[i + 1][j];
                }
            }
        }  
        for (int i = 0; i < row-1; i++)
        {
            for (int j = 0; j < banyak_row1; j++)
            {
                printf("%d ", elemen[i][j]);
            }
        }
    }   
    else{
    printf("Jumlah tidak sama");
    }
    return 0;
}